vita3k/gui/src/license_install_dialog.cpp

```diff
@@ -39,6 +35,20 @@ void draw_license_install_dialog(GuiState &gui, EmuEnvState &emuenv) {
     auto &indicator = gui.lang.indicator;
     auto &common = emuenv.common_dialog.lang.common;
 
+    enum class State {
+        UNDEFINED,
+        LICENSE,
+        ZRIF,
+        SUCCESS,
+        FAIL
+    };
+
+    static State state = State::UNDEFINED;
+
+    static std::string title, zRIF;
+    static std::filesystem::path license_path = "";
+    static bool delete_license_file;
```

Comment:
From its use, there is no reason to make delete_license_file a static

Reply:
is neded here i thinks, if not when you check box is disable directly ? 🤔

Reply:
if `delete_license_file` is not static then checkbox does not save its state between frames, so it's impossible to set checkbox on.

Reply:
Yes is what i said